source/quick-start.txt

```diff
@@ -157,7 +157,7 @@ instructions on how to format your connection string.
 
 To retrieve your connection string for the instance and user you created in
 the previous step, log into your Atlas account and navigate to the
-**Clusters** section and click the **Connect** button for the cluster that you
+**Database** section and click the **Connect** button for the cluster that you
```

Comment:
Issue: the references to the GUI elements should use `:guilabel:` per the style guide.